protocol/flows/ping/receive.go

```diff
@@ -0,0 +1,40 @@
+package ping
+
+import (
+	"github.com/kaspanet/kaspad/netadapter/router"
+	"github.com/kaspanet/kaspad/wire"
+)
+
+// ReceivePingsContext is the interface for the context needed for the ReceivePings flow.
+type ReceivePingsContext interface {
+}
+
+type receivePingsFlow struct {
+	incomingRoute, outgoingRoute *router.Route
```

Comment:
I think you want to embed the context into the flow struct.
Since we introduced the context to "comply to the standard" - let's comply with the standard all the way.